All chart stuff

gdoyle1 opened the pull request:
Close #720 #747 

Added the chart color system to the styles section and added charting design guidelines

gdoyle1 commented:
@mcarrano Changes have been made! I spoke with Stacy and she said H2/H3s are fine as they are now. Let me know if you need me to do anything else

mcarrano commented on packages/patternfly-4/content/design-guidelines/usage-and-behavior/Charts for colors/design.md:
```diff
@@ -0,0 +1,115 @@
+path: "/design-guidelines/usage-and-behavior/charting"
+
+# Charts for colors
```
I think you mean "Colors for charts"? :)

gdoyle1 commented:
@mcarrano Just fixed the typos. I renamed the usage-and-behavior guidelines section to "charts" and then added everything back into that folder (.md and imgs folder). I added "charts" to the usage-and-behavior part of the navigation.

gdoyle1 commented:
@mcarrano Okay this should now be ready to go!

mindreeper2420 commented on packages/patternfly-4/content/design-guidelines/styles/charting_color_system.md:
```diff
@@ -0,0 +1,39 @@
+path: /design-guidelines/styles/charting-color-system
```
@gdoyle1 the path definition needs to be bracketed with `---` at top and bottom. Currently, the site cannot locate this page. You also need quotations around the path.

```
---
path: "/design-guidelines/styles/charting-color-system"
---
```

gdoyle1 replied:
how do I delete this? It's only supposed to be a path for "charts" in design-guidelines and then "colors-for-charts" in styles

mindreeper2420 commented on packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md:
```diff
@@ -0,0 +1,39 @@
+path: /design-guidelines/styles/colors-for-charts
```
same thing here:

```
---
path: "/design-guidelines/styles/colors-for-charts"
---
```

mindreeper2420 commented on packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md:
```diff
@@ -1,4 +1,6 @@
-path: /design-guidelines/styles/colors-for-charts
+---
+path: ""/design-guidelines/styles/colors-for-charts"
```
@gdoyle1  The extra `"` at the beginning of the path needs to be removed.

mindreeper2420 commented on packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md:
```diff
@@ -0,0 +1,41 @@
+---
```
@gdoyle1 Here is the block of code to use at the top of the page for referencing the GIFs:

```
---
path: "/design-guidelines/styles/colors-for-charts"
---
import theSystem from './thesystem.gif';
import oneFamily from './onefamily.gif';
```

mindreeper2420 commented on packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md:
```diff
@@ -0,0 +1,41 @@
+## The system
+Although the system may seem complex, it is completely customizable. The idea is that you can put together your desired colors and amount of colors and the system will still sort them correctly. Here’s how the system works:
+
+![The system gif](thesystem.gif)
```
Change this to

```
<img src={theSystem} />
```

mindreeper2420 commented on packages/patternfly-4/content/design-guidelines/styles/colors-for-charts.md:
```diff
@@ -0,0 +1,41 @@
+
+You can also use the color order system with one color family. This creates a monochromatic effect. It works using the same logic and order. We recommend using this system for charts with nested variables. The order begins by going through each base color, then lightest, darkest, second lightest, and ending with second darkest. In variable-terms: #1 [color]-300 (base color), #2 [color]-100, #3 [c ...
+
+![One color family gif](onefamily.gif)
```
Change this to:

```
<img src={oneFamily} />
```
